src/gossip/active_set.zig

```diff
@@ -22,40 +22,40 @@ const BLOOM_MAX_BITS: usize = 1024 * 8 * 4;
 
 pub const ActiveSet = struct {
     // store pubkeys as keys in gossip table bc the data can change
-    pruned_peers: std.AutoHashMap(Pubkey, Bloom),
+    peers: std.AutoHashMap(Pubkey, Bloom),
```

Comment:
can you add a comment mentioning how the bloom tracks origins which have been pruned from peers that we push to above this field?

Reply:
Yeah cool ill add that, on another note I'm not quite sure what the "// store pubkeys..." comment is getting at? Could you fill me in on that?

Reply:
i think a previous approach would store the full contact info's in the hashmap to reduce looking up the value in the gossip table - but since the values in the gossip table can change, we only store the pubkey and do a lookup when we need to read their contact info